### PR TITLE
habana dlc 1.6.0 release

### PR DESCRIPTION
**Pull request by bukeao**
*GitHub Issue #, if available:*

Note: If merging this PR should also close the associated Issue, please also add that Issue # to the Linked Issues section on the right. 

### Description

Adding Habana DLC support for SynapseAI v1.6.0 for PT1.12 and TF2.9

### Tests run
**NOTE: If you are creating a PR for a new framework version, please ensure success of the standard, rc, and efa sagemaker remote tests by updating the dlc_developer_config.toml file:**

- [ ] Revision A: `sagemaker_remote_tests = "standard"`
- [ ] Revision B: `sagemaker_remote_tests = "rc"`
- [ ] Revision C: `sagemaker_remote_tests = "efa"`

**Additionally, please run the sagemaker local tests in at least one revision:**
- [ ] `sagemaker_local_tests = true`

## Label Checklist
- [ ] I have added the project label for this PR (*<project_name>* or "Improvement")

## PR Checklist
- [ ] I've prepended PR tag with frameworks/job this applies to : [mxnet, tensorflow, pytorch] | [ei/neuron/graviton] | [build] | [test] | [benchmark] | [ec2, ecs, eks, sagemaker]
- [ ] If the PR changes affects SM test, I've modified dlc_developer_config.toml in my PR branch by setting sagemaker_tests = true and efa_tests = true
- [ ] If this PR changes existing code, the change fully backward compatible with pre-existing code. (Non backward-compatible changes need special approval.)
- [ ] (If applicable) I've documented below the DLC image/dockerfile this relates to
- [ ] (If applicable) I've documented below the tests I've run on the DLC image
- [ ] (If applicable) I've reviewed the licenses of updated and new binaries and their dependencies to make sure all licenses are on the Apache Software Foundation Third Party License Policy Category A or Category B license list.  See [https://www.apache.org/legal/resolved.html](https://www.apache.org/legal/resolved.html).
- [ ] (If applicable) I've scanned the updated and new binaries to make sure they do not have vulnerabilities associated with them.

## Pytest Marker Checklist
- [ ] (If applicable) I have added the marker `@pytest.mark.model("<model-type>")` to the new tests which I have added, to specify the Deep Learning model that is used in the test (use `"N/A"` if the test doesn't use a model)
- [ ] (If applicable) I have added the marker `@pytest.mark.integration("<feature-being-tested>")` to the new tests which I have added, to specify the feature that will be tested
- [ ] (If applicable) I have added the marker `@pytest.mark.multinode(<integer-num-nodes>)` to the new tests which I have added, to specify the number of nodes used on a multi-node test
- [ ] (If applicable) I have added the marker `@pytest.mark.processor(<"cpu"/"gpu"/"eia"/"neuron">)` to the new tests which I have added, if a test is specifically applicable to only one processor type

#### EIA/NEURON/GRAVITON Testing Checklist
* When creating a PR:
- [ ] I've modified `dlc_developer_config.toml` in my PR branch by setting `ei_mode = true`, `neuron_mode = true` or `graviton_mode = true`

#### Benchmark Testing Checklist
* When creating a PR:
- [ ] I've modified `dlc_developer_config.toml` in my PR branch by setting `benchmark_mode = true`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: habana/pytorch/training/docker/1.12/py3/synapseai1.6.0/Dockerfile.hpu [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    werkzeug \
+ && pip install --no-cache-dir --user -U boto3 --upgrade \
+ && pip uninstall -y torch \
+ && pip uninstall -y hb-torch \
+ && pip install --no-cache-dir --user -U ${PT_TRAINING_URL} \
+ && rm -rf ${PT_TRAINING_URL}
+
+ENV LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/root/.local/lib
+
+#RUN ln -s $(which ${PYTHON}) /usr/local/bin/python \
```
Comment (tejaschumbalkar) [inline habana/pytorch/training/docker/1.12/py3/synapseai1.6.0/Dockerfile.hpu:65]: why is this commented?

File: habana/tensorflow/training/docker/2.9/py3/synapseai1.6.0/Dockerfile.hpu [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    wget \
+    vim \
+    zlib1g-dev \
+    # Install dependent library for OpenCV
+    libgtk2.0-dev \
+ && rm -rf /var/lib/apt/lists/* \
+ && apt-get clean
+
+# SSH login fix. Otherwise user is kicked off after login
+RUN sed 's@session\s*required\s*pam_loginuid.so@session optional pam_loginuid.so@g' -i /etc/pam.d/sshd
```
Comment (hballuru) [inline habana/tensorflow/training/docker/2.9/py3/synapseai1.6.0/Dockerfile.hpu:55]: Just a minor observation and not a concern for this PR. But just curious... Is this SSH login fix not required/needed for the PyTorch image? I see this directive only on the Tensorflow image and not on the other one.

File: habana/pytorch/training/docker/1.12/py3/synapseai1.6.0/Dockerfile.hpu (hunk 1 above)
Comment (tejaschumbalkar) [reply habana/pytorch/training/docker/1.12/py3/synapseai1.6.0/Dockerfile.hpu:65]: @bukeao ^^

Comment (bukeao) [reply habana/pytorch/training/docker/1.12/py3/synapseai1.6.0/Dockerfile.hpu:65]: it prints: 
ln: failed to create symbolic link '/usr/bin/pip'
: File exists

File: habana/tensorflow/training/docker/2.9/py3/synapseai1.6.0/Dockerfile.hpu (hunk 2 above)
Comment (bukeao) [reply habana/tensorflow/training/docker/2.9/py3/synapseai1.6.0/Dockerfile.hpu:55]: the change is not from us, I checked the code, it looks like added by Tejas Chumbalkar #1478, I don't have access to the commit.

Comment (hballuru) [reply habana/tensorflow/training/docker/2.9/py3/synapseai1.6.0/Dockerfile.hpu:55]: got it. thanks

Comment (tejaschumbalkar) [reply habana/tensorflow/training/docker/2.9/py3/synapseai1.6.0/Dockerfile.hpu:55]: this file was sourced into the public repo by me and hence my name on the commit. In any case, it seems to be there from the start v0.15.4, lets keep it

File: dlc_developer_config.toml [hunk 3]
```diff
@@ -12,20 +12,20 @@ neuron_mode = false
 graviton_mode = false
 # Please only set it to True if you are preparing a HABANA related PR
 # Do remember to revert it back to False before merging any PR (including HABANA dedicated PR)
```
Comment (tejaschumbalkar) [inline dlc_developer_config.toml:14]: Revert the changes in this file to prepare for merge

**Review by shantanutrip — COMMENTED** [review]
The PR looks good. Lets revert the temp changes and get it merged.

**Comment by tejaschumbalkar** [comment]
Please update the PR description

**Comment by bukeao** [comment]
> Please update the PR description

what changes do you need

**Comment by tejaschumbalkar** [comment]
> Description

Please update the `Description` section

**Review by tejaschumbalkar — APPROVED** [review]
